deploy/kustomize/testing/controller-coverage-patch.yaml

```diff
@@ -14,3 +14,24 @@
     hostPath:
       path: /var/lib/pmem-csi-coverage
       type: DirectoryOrCreate
+# driver in controller mode runs as non-root user
+# which cannot have enough permissions to access
+# coverage-dir(/var/lib/pmem-csi-coverage)
+# Change the ownership of this folder to "pmem-csi:pmem-csi"
+# so that driver could write coverage reports.
+# NOTE: The assumption is that the user,group exist on the VM host with id 1000 
```

Comment:
Why this assumption? Files can be owned by numeric IDs that don't have entries in /etc/passwd or /etc/groups.

Reply:
> Why this assumption? Files can be owned by numeric IDs that don't have entries in /etc/passwd or /etc/groups.

Ok, this I do not know. I will remove the note.